src/experiences.ts

```diff
@@ -48,10 +48,10 @@ export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience
 }
 
 const experiencesArray: Experience[] = [
-    { api: API.DocumentDB, longName: "SQL", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB },
+    { api: API.DocumentDB, longName: "SQL", description: "(DocumentDB)", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB },
```

Comment:
'SQL (DocumentDB)' in one place and 'DocumentDB (SQL)' everywhere else seems a bit confusing to the user. 
For the sake of consistency, I think the order here should be swapped, considering what's done in the other PR.

Reply:
I agree with @PrashanthCorp that we should be consistent, but I prefer 'SQL (DocumentDB)' so this PR looks fine to me

Reply:
I'll change the other PR.